django_hashedfilenamestorage/tests.py

```diff
@@ -110,16 +110,16 @@ def test_save(self):
             name1 = storage.save('foo/bar.txt', ContentFile(self.CONTENT))
             self.assertEqual(name1,
                              'foo/%s.txt' % self.SHA1SUM)
-            self.assertEqual(storage.open(name1).read(), self.CONTENT)
+            self.assertEqual(storage.open(name1, 'rt').read(), self.CONTENT)
```

Comment:
Is `'rt'` not the default for the storage already in Python 3?

Reply:
Yes, but the test fails under python 2 if it isn't specified

Reply:
Ah, of course, cheers for reminder =)